README.md

```diff
@@ -30,6 +30,18 @@ action "Alert" {
 
 The next time anyone pushes to the repository, if any of the commits in the push modify `db/structure.sql`, the `Check` Action will pass, triggering the `Alert` Action, which will send you a message in Slack. If none of the commits in the push modify `db/structure.sql`, the `Check` Action will fail, and the `Alert` Action will not be triggered.
 
+### Arguments
+
+The argument is the path to one or more files. The action will pass if any of the matched files is modified.
+
+[Glob patterns](https://en.wikipedia.org/wiki/Glob_(programming)) can also be used.
```

Comment:
Love the link! What do you think about including a link to the [Ruby-specific docs](https://ruby-doc.org/core-2.6.3/File.html#method-c-fnmatch) instead of, or in addition to, the Wikipedia article?

Reply:
Added link to documentation in 720330609f0808a1592396b17074d4625fd2968b.